DotNetRu.Clients.Portable/ViewModel/EventsViewModel.cs

```diff
@@ -137,7 +137,7 @@ public void SortEvents()
                 this.Events?.ReplaceRange(GetEvents());
                 //Events.ReplaceRange(await StoreManager.EventStore.GetItemsAsync(force));
 
-                this.Title = "Meetups (" + this.Events?.Count(e => e.StartTime.HasValue) + ")";
```

Comment:
Please exclude it - I've already changed Title to display total number of meetups

Reply:
But why we need to show total number of meetups? For me - it's useless

Reply:
Ok, I agree now